lib/rules/void-dom-elements-no-children.js

```diff
@@ -50,6 +50,7 @@ function errorMessage(elementName) {
 
 module.exports = {
   meta: {
+    type: 'suggestion',
```

Comment:
```suggestion
    type: 'problem',
```
this is always a bug

Reply:
Change category to "Possible errors"?